org.squidowl.halloy.json

```diff
@@ -0,0 +1,61 @@
+{
+    "app-id": "org.squidowl.halloy",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "22.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "halloy",
+    "finish-args": [
+        "--share=ipc",
+        "--filesystem=xdg-config",
+        "--filesystem=xdg-data",
```

Comment:
why do you need access to these?

Reply:
Use config and state files are persisted here. We're using `HOST_XDG_*` vars to access the host XDG dirs.

Reply:
if you are XDG compliant you don't need either.

Reply:
I'm not sure I'm following. We prefer to use the host's XDG paths vs `~/.var/app/..`. Unless you're suggesting something else.

Reply:
you are not indeed. You don't need to set any of these permissions, which are excessive, for this to work.

Reply:
I don't see how this is correct. I just removed those and it prevent us accessing those locations `~/.config/halloy/` / `~/.local/share/halloy` on my system.

We are not looking to force users into managing their dotfiles for this application in the `~/.var` location. I'd much rather respect what their host XDG directories are, which these permissions give us access to.